scripts/module-settings.js

```diff
@@ -291,7 +318,7 @@ Hooks.once('ready', () => {
     return spell_names;
   });
 
-  Handlebars.registerHelper('me-HTMLtoYAML', function (text, context, options) {
+  const htmlToYaml = (text, context, options) => {
```

Comment:
I'm curious why this is changed to registered this particular handlebar helper differently.   I'm very new to javascript and handlebars (this project has been my introduction to it) so I'm assuming I have done a lot of bad practices along the way.

Reply:
Is there an advantage to declaring the function independently and then registering it as a helper later?